### DISCUSSION TITLE
Create groups for elements before being pushed to ETABS

### DISCUSSION THREAD

**Discussion by kevinbajracharya** [BHoM/BHoM#1605] [Ideas]
Would it be possible to add a feature that can assign groups to the elements before being pushed into ETABS?
As an example shown in the image below.
![image](https://github.com/BHoM/BHoM/assets/114489103/b44abce3-1eb1-4d79-b8ad-3bd25d7827bd)

We can utilise the group properties already available in ETABS
![image](https://github.com/BHoM/BHoM/assets/114489103/cb0954c0-7ba9-406a-bd71-c333adf812bf)

This would greatly benefit projects where one has a large number of different elements with the same section properties.

**Comment by adityaramboll** [comment]
@kevinbajracharya  In the Robot Toolkit there is implementation for adding groups https://github.com/BHoM/Robot_Toolkit/tree/develop/Robot_Adapter/CRUD/Create/Groups%20and%20Lists
It can be a good reference to create your own for ETABS

**Comment by samuelbasimalla** [comment]
This feature for adding groups is now incorporated in ETABS toolkit. 

Reference PR:
https://github.com/BHoM/ETABS_Toolkit/pull/516
